terraform-checksums.json

```diff
@@ -923,20 +923,6 @@
       "binary_checksum": "bac6bb1fb9383249b40afa46b83a4a78fa9f581347195f0e12bd1967d14a4823",
       "os": "linux",
       "arch": "arm64"
-    },
```

Comment:
This is removed to reset the release process, I will manually run it again to generate a new 0.3.2 release.